src/freenas/usr/local/lib/middlewared_truenas/alert/source/failover_sync.py

```diff
@@ -32,3 +32,17 @@ class FailoverKeysSyncFailedAlertClass(AlertClass, SimpleOneShotAlertClass):
     )
 
     products = ("ENTERPRISE",)
+
+
+class FailoverKMIPKeysSyncFailedAlertClass(AlertClass, SimpleOneShotAlertClass):
+    deleted_automatically = False
+
+    category = AlertCategory.HARDWARE
+    level = AlertLevel.CRITICAL
+    title = "Syncing KMIP Keys to Peer Failed"
+    text = (
+        "The automatic synchronization of KMIP keys with the standby "
```

Comment:
Since you're here, can we standardize the "storage controller" verbiage. Product management determined that it needs to be `passive storage controller`. Can you update all the one-shot alerts in this file to be the same?

Reply:
Sure ;) Please feel free to review the complete PR, I'm just waiting for it to QA with TC cloud : )

Reply:
Will do 👍